CoreWebServer/src/au/gov/asd/tac/constellation/webserver/api/SwaggerServlet.java

```diff
@@ -122,20 +124,31 @@ protected void doGet(final HttpServletRequest request, final HttpServletResponse
                     final ArrayNode params = httpMethod.putArray("parameters");
                     rs.createParameters().getParameters().entrySet().forEach(entry -> {
                         final PluginParameter<?> pp = entry.getValue();
-                        final ObjectNode param = params.addObject();
-                        param.put("name", pp.getId());
-                        param.put("in", pp.getName().toLowerCase(Locale.US).contains("(body)") ? "body" : "query");
-                        param.put("required", false); // TODO Hard-code this until PluginParameters grows a required field.
-                        param.put(DESCRIPTION, pp.getDescription());
-                        final ObjectNode schema = param.putObject(SCHEMA);
-                        schema.put("type", pp.getType().getId());
+
+                        if (pp.getName().toLowerCase(Locale.ENGLISH).contains("(body)")) {
+                            final ObjectNode requestBody = httpMethod.putObject("requestBody");
+                            requestBody.put(DESCRIPTION, pp.getName().replace("(body)", " - ") + pp.getDescription());
+                            requestBody.put(REQUIRED, false); //fix in the other ticket
```

Comment:
@Auriga2 can you please prefix this comment with "TODO" and you can mention the GitHub issue like #123. Thanks.

Reply:
Fixed.